### DISCUSSION TITLE
[Feature] search by filename / originalPath (using pattern matching - LIKE, SIMILAR)

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6276] [Feature Request]
### The feature

I would like to be able to search by originalPath (containing also the original filename)
This is especially useful in external libraries, which are organized in a hierarchical tree.
https://www.postgresql.org/docs/current/functions-matching.html should be used (LIKE, SIMILAR)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by v411e** [comment]
This is working again since 1.93.x :tada: 
(See https://github.com/immich-app/immich/issues/5982)

**Reply by aisbergde** [reply]
If it was possible, why it has been removed? Or will it be possible again in the future, when the search will be rebuild in general?

**Reply by alextran1502** [reply]
@aisbergde It was possible with Typesense. Now that we removed Typesense that mechanism is temporarily gone with it. We will add it back in the search enhancement work

**Comment by frigi83** [comment]
There are also 2 open issues: #5982 and #6056
